source/install/prod-windows-2012.rst

```diff
@@ -0,0 +1,419 @@
+..  _prod-windows:
+
+Production Install on Windows Server
+==========================================
+
+.. attention :: This install guide has been contributed by the Mattermost community. 
+   It has not yet been tested by the core team. We have `an open ticket 
+   <https://github.com/mattermost/docs/issues/360>`__ requesting community help testing and 
+   improving this guide. Once the community has confirmed we have multiple deployments on these 
```

Comment:
`requesting community help testing and improving this guide` --> `requesting community help test and improve this guide`